src/plugins/blocksmith/components/blocksmith/Blocksmith.jsx

```diff
@@ -1197,6 +1197,7 @@ class Blocksmith extends React.Component {
     this.setState(() => ({
       modalComponent,
     }));
+    document.querySelector('.inline-toolbar').style.visibility = 'hidden';
```

Comment:
@meghedi do it the react way with conditional rendering - just dont include the toolbar at all in the dom when the modal is open

Reply:
@efjacobson I tried using react conditions like the change that I commented out, but still as soon as the modal is closed the toolbar shows up on the bottom. but with styling it does not. still working to find a way, I can show you when you have time.

Reply:
@meghedi looks like there is an isVisible prop that it can take, try that https://github.com/draft-js-plugins/draft-js-plugins/blob/master/packages/inline-toolbar/src/components/Toolbar/index.tsx#L32